src/commands/backend/list.ts

```diff
@@ -0,0 +1,49 @@
+import { Command, CliUx } from '@oclif/core'
+import { BackendWithStatus } from '../../api'
+import { Jamsocket } from '../../jamsocket'
+
+function formatDate(date: string): string {
+  const d = new Date(date)
+  const tzOffset = d.getTimezoneOffset()
```

Comment:
If the date is coming from the DB, I think we always store it as UTC? Maybe instead of GMT+0, we could just say UDT.

Reply:
This ends up producing the timezone offset of the user's machine. Our API returns timestamps like this: `2023-03-31T01:40:28.087054Z`. When you pass that into a JavaScript `Date` function and then render in the user's local time, it leaves off the timezone offset, so it's not clear to the reader if this is UTC or local time. Other timestamps I've seen render with a `GMT+HHMM` at the end, which is what I was trying to copy.

```js
const timestamp = '2023-03-31T01:40:28.087054Z'
const d = new Date(timestamp)
d.toLocaleString() // '3/30/2023, 9:40:28 PM'
d.toTimeString() // '21:40:28 GMT-0400 (Eastern Daylight Time)'
formatDate(d) // '3/30/2023 9:40:28 PM (GMT-0400)'
```

Maybe I'll just use `formatDistanceToNow`, though. (Also, there's a bug in my `formatDate` that sticks a semicolon between HH and MM and flips the tzoffset sign.)